x/virtualgroup/keeper/keeper.go

```diff
@@ -358,6 +369,11 @@ func (k Keeper) SwapAsPrimarySP(ctx sdk.Context, primarySP, successorSP *sptypes
 	}
 	k.SetGVGStatisticsWithSP(ctx, srcStat)
 	k.SetGVGStatisticsWithSP(ctx, dstStat)
+	if ctx.IsUpgraded(upgradetypes.Serengeti) {
+		k.DeleteGVGFamilyWithinSP(ctx, srcVGFStat.SpId, family.Id)
```

Comment:
can you rename this, a little confused..

Reply:
rename to DeleteSpecificGVGFamilyStatisticsFromSP. If it is not appropriate, i will think about another one